site/docs/v1/tech/apis/form-fields/_form-field-type.adoc

```diff
@@ -13,4 +13,9 @@ When using the `date` type, the only supported data format is ISO-8601 or `YYY-M
 +
 When using the `consent` type, the field [field]#consentId# must be included in the request.
 +
-Defaults to `string`, unless `user.birthDate`, then `date`, or if `user.email` then `email`.
\ No newline at end of file
+Defaults can change depending on the type of `key`. The defaults are:
```

Comment:
Since we do have a `type` field, we should probably describe this as the `on the value of the key`. Same for above description of form control defaults.

This should be a field reference `[field]#key#`.